src/angular/controllers/ctrl-widget-catalog-details.js

```diff
@@ -0,0 +1,196 @@
+const app = angular.module('materia')
```

Comment:
Since the widget catalog and details pages have been completely rewritten in React, this file is no longer necessary. I'll make a separate commit to remove it.